src/test/java/rv/atqc/gttesting/editmyprofiledata/po/US_EditMyProfileDataWithChrome.java

```diff
@@ -0,0 +1,154 @@
+package rv.atqc.gttesting.editmyprofiledata.po;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+import rv.atqc.gttesting.archex.EditProfilePage;
+import rv.atqc.gttesting.archex.HeadForGuestUserPage;
+import rv.atqc.gttesting.archex.ProfilePage;
+import rv.atqc.gttesting.res.Resources;
+
+public class US_EditMyProfileDataWithChrome {
+
+	private WebDriver driver;
+	private ProfilePage pageProfile;
+	private EditProfilePage page;
+
+	@BeforeMethod
+	public void before() {
+		ChromeDriverManager.getInstance().setup();
+		driver = new ChromeDriver();
+		driver.get(" http://green-tourism.herokuapp.com");
+		driver.manage().window().maximize();
+		pageProfile = new HeadForGuestUserPage(driver).clickLeftNavButton().clickLogIn()
+				.setLoginField(Resources.LogInMessage.USER_EMAIL).setPasswordField(Resources.LogInMessage.USER_PASSWORD).clickLogInButton().clickLeftNavButton()
```

Comment:
I would prefer to move Resources.LogInMessage to the private field in order to avoid useless calls to the Resources.LogInMessage. For example:

private LogInMessage = Resources.LogInMessage;

.....

@BeforeMethod
 	public void before() {
                ....... 		
 		pageProfile = new HeadForGuestUserPage(driver).clickLeftNavButton().clickLogIn()		.setLoginField(Resources.LogInMessage.USER_EMAIL).setPasswordField(LogInMessage.USER_PASSWORD).clickLogInButton().clickLeftNavButton()
                .....
}